setup.cfg

```diff
@@ -7,8 +7,8 @@ all_files = 1
 upload-dir = doc/_build/html
 show-response = 1
 
-[pytest]
-minversion = 2.2
+[tool:pytest]
+minversion = 3
```

Comment:
@pllim, astropy-helpers no longer includes its own version of `pytest` right?  I am ok with switching from the default `unittest` to `pytest`, but is this declaration here sufficient to make it a requirement, or does it become a "recommended" 
 install?

Reply:
Yes, astropy removed pytest from its bundled packages. The specified version is the *minimal required* version; see https://docs.pytest.org/en/latest/customize.html#confval-minversion